docs/real-time/azure-signalr-scenario.md

```diff
@@ -245,13 +247,13 @@ dotnet add package Microsoft.Azure.SignalR.Management
 
 ---
 
-Azure SignalR _Serverless_ mode doesn't require a hub server to be running, which is why this mode is named "serverless". The Azure SignalR Service is responsible for maintaining client connections. Consider the following diagram that illustrates the architecture of Azure SignalR Service in _Serverless_ mode:
+Azure SignalR _Serverless_ mode doesn't require a hub server to be running, which is why this mode is named "serverless". The Azure SignalR Service is responsible for maintaining client connections. Additionally, in this mode, your limited to broadcast only—meaning that messages can be broadcast from a `ServiceHubContext` but not a traditional <xref:Microsoft.AspNetCore.SignalR.Hub>, <xref:Microsoft.AspNetCore.SignalR.Hub`1>, or <xref:Microsoft.AspNetCore.SignalR.IHubContext`1>. Consider the following diagram that illustrates the architecture of Azure SignalR Service in _Serverless_ mode:
```

Comment:
`in this mode, your limited to broadcast only` - this is not accurate. In serverless mode, user could configure [upstream endpoint](https://learn.microsoft.com/en-us/azure/azure-signalr/concept-upstream) so that the upstream could handle messages from the clients. The upstream is usually a Azure Function SignalR trigger. Having `SignalR Client<->Azure SignalR emulator <-> Azure Function` works locally, however deploying to cloud Azure Function needs some additional work because Azure Function requires an app key code, e.g. `<endpoint>/runtime/webhooks/signalr/?code=xxx`, and that's why we don't use Azure Function as the sample.